### PR TITLE
Implemented properly streams closure after running cmd in JavaScript

### LINKED ISSUE #1591: Proper streams closure after running cmd in JavaScript
**Description**

Figure out how to properly close input and error stream after running cmd.
Currently, the closure of the streams is not controlled, which can lead to a loss of resources.

**To Reproduce**

1. Open `utbot-js/../utils/JsCmdExec.kt`
2. Go to usages of `runCommand` function
3. Look at the operations with the returned `bufferReader`

**Expected behavior**

After executing the cmd command, the inputStream and errorStream text should be obtained, both streams should be closed properly.

**Actual behavior**

The closure of streams is not controlled properly.

**Potential problem solution**

Close the streams in `runCommand` function and return a `Pair<String, String>`

```Kotlin
fun runCommand(
        dir: String? = null,
        shouldWait: Boolean = false,
        timeout: Long = defaultTimeout,
        vararg cmd: String,
    ): Pair<String, String> {
        val builder = ProcessBuilder(*OsProvider.getProviderByOs().getCmdPrefix(), *cmd)
        dir?.let {
            builder.directory(File(it))
        }
        val process = builder.start()
        if (shouldWait) {
            if (!process.waitFor(timeout, TimeUnit.SECONDS)) {
                process.descendants().forEach {
                    it.destroy()
                }
                process.destroy()
                throw TimeoutException("")
            }
        }
        return Pair(
            process.inputStream.bufferedReader().use { it.readText() },
            process.errorStream.bufferedReader().use { it.readText() }
        )
    }
```

### PR DESCRIPTION
**Pull request by rudolf101**
# Description

After running cmd execution for JS, the received `inputStream` and `errorStream` are closed properly.

Fixes #1591 

## Type of Change

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Manually

# Checklist:

- [x] The change followed the style guidelines of the UTBot project
- [x] Self-review of the code is passed
- [x] No new warnings
- [x] All tests pass locally with my changes

### CODE REVIEW

**Comment by alisevych** [comment]
@rudolf101 @denis-fokin Could you please add corresponding component label? (comp-...)

File: utbot-cli-js/src/main/kotlin/org/utbot/cli/js/JsRunTestsCommand.kt [hunk 1]
```diff
@@ -39,22 +39,21 @@ class JsRunTestsCommand : CliktCommand(name = "run_js", help = "Runs tests for t
...
                     logger.error { "An error has occurred while running tests for $fileWithTests : $errorText" }
                 } else {
-                    val text = textReader.readText()
                     outputAbsolutePath?.let {
                         val file = File(it)
                         file.createNewFile()
-                        file.writeText(text)
-                    } ?: logger.info { "\n$text" }
+                        file.writeText(inputText)
+                    } ?: logger.info { "\n$inputText" }
```
Comment (denis-fokin) [inline utbot-cli-js/src/main/kotlin/org/utbot/cli/js/JsRunTestsCommand.kt:54]: Would not be a line separator a better solution?

File: utbot-js/src/main/kotlin/service/FastCoverageService.kt [hunk 2]
```diff
@@ -141,9 +141,8 @@ class FastCoverageService(
                     coverageList.add(index to json.getJSONObject("s"))
                     _resultList.add(index to json.get("result").toString())
                 }
-                val errText = error.readText()
-                if (errText.isNotEmpty()) {
-                    println(errText)
+                if (errorText.isNotEmpty()) {
+                    println(errorText)
```
Comment (denis-fokin) [inline utbot-js/src/main/kotlin/service/FastCoverageService.kt:145]: Why not error log?

**Comment by rudolf101** [comment]
Fixed remarks, ready for review

**Pull request closed without merging**